common/test/acceptance/pages/lms/video/video.py

```diff
@@ -356,9 +356,9 @@ def click_player_button(self, button):
 
         self.q(css=button_selector).first.click()
 
-        button_states = {'play': 'playing', 'pause': 'pause'}
-        if button in button_states:
-            self.wait_for_state(button_states[button])
+        # button_states = {'play': 'playing', 'pause': 'pause'}
+        # if button in button_states:
+        #     self.wait_for_state(button_states[button])
```

Comment:
@polesye Could there be a bug where a class (is-paused) is not getting removed? The state property returns a list, and the state method is [returning](https://github.com/edx/edx-platform/blob/master/common/test/acceptance/pages/lms/video/video.py#L674) with the first match it finds in the list.

Unrelated to this particular testcase, but I think there is at least one test with 2 videos on the page. Looking at the code, wait_for_state uses self.state, which [always uses the first video](https://github.com/edx/edx-platform/blob/master/common/test/acceptance/pages/lms/video/video.py#L664).